Telegram bot

```diff
@@ -0,0 +1,256 @@
+import numpy as np
+import math
+import telebot
+from telebot import types
+from peewee import *
+from datetime import datetime
+import threading, schedule
```

Comment:
I suppose it's better to regroup imports – at first, all `import ...`, then all `from ... import ...`